app/navigation/Navigator.scala

```diff
@@ -439,6 +458,8 @@ class Navigator @Inject() (categorisationService: CategorisationService) {
     case HasNirmsUpdatePage                        => navigateFromHasNirmsUpdate
     case HasNiphlPage                              => navigateFromHasNiphlCheck
     case NiphlNumberPage                           => _ => routes.CyaCreateProfileController.onPageLoad
+    case RemoveNiphlPage                           => navigateFromRemoveNiphlPage
```

Comment:
i think this should be for the 'check' version of these functions (see other examples in this match)

Reply:
we use the same method for both normal and check mode, i can create a navigateFromRemoveNiphlPagecheck method but it would be the same code?

Reply:
hmm good point - if mode doesn't matter why do we enter into the check version of this at all?

i guess if you check other examples there is always a check even if it is the same code? I'm a big fan of following the pattern that is already there - or changing it all to a new pattern if the old pattern is bad, it feels a little risky to have one pattern for one way but another pattern for another way

Reply:
I assume to a) Because the page we are on has a checkmode, but where we go doesn't depend on that.

Which kinda doesn't sound right to me tbh. Not sure we should be breaking out the journey if we were at CYA to go back to Profile Controller.

Reply:
i feel like remove niphl shouldn't have a 'check' url

Reply:
RemoveNiphl doesnt have check mode so ive removed this now